meson_options.txt

```diff
@@ -0,0 +1,7 @@
+#===============
+# custom options
+#===============
+
+option('libcpp_clang',     type : 'boolean', value : false, description : 'use libc++ if compiling with clang and the library is available')
+option('test',             type : 'boolean', value : false, description : 'activate tests (debug only)')
+option('download_missing', type : 'combo',  choices: ['minimal', 'full'], value : 'minimal', description : 'activate tests (debug only)')
```

Comment:
What does this option do? Its description is based on the description of the "test" option...

(Also: github is complaining about the newline ending of the file.)

Reply:
actually nothing, it was supposed to be used to activate tests, however since there is already an option to do so, it will be removed.
download_missing will be used (and renamed) to activate system-only usage of heavy libraries.